_docs/style/text-conventions/general-guidelines.md

```diff
@@ -0,0 +1,194 @@
+
+{% figure [caption:"Provide context example"] [class:"image bg-light border"] %}
+| Use | Avoid  |
+| --- | --- |
+| This stack includes a Cloud Load Balancer, a Cloud Database, and a Master server (plus optional secondary servers). | This stack includes a Cloud Load Balancer, Cloud Database, and a Master server (plus optional secondary servers). |
+{: class="hxTable"}
+{% endfigure %}
+
+In the correct example, notice the **a** before Cloud Database.
+
+Parallel structure also applies to options you present to users. Don't ask users to choose from unrelated values. For example, when providing radio button options:
+
+ {% figure [caption:"Provide context example"] [class:"image bg-light border"] %}
+ | Use | Avoid  |
+ | --- | --- |
+ | **Always**, **Only with streaming**, and **Never**. | **Always**, **Only with streaming**, and **Blocked**. |
+ {: class="hxTable"}
+ {% endfigure %}
+
+In this example, **Always** and **Never** are opposites. **Always** and **Blocked** are not opposites.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+## Provide context
+
+Provide meaningful descriptions for key features and state their importance to help users understand the benefits. Don't use Rackspace-created terms as a substitute for clearly explaining options.
+
+{% figure [caption:"Provide context example"] [class:"image bg-light border"] %}
+| Use | Avoid  |
+| --- | --- |
+| Monitoring keeps your applications up and running. You can use monitoring to track and gather information about the applications that you run to prevent downtime. | Use Rackspace Monitoring to prevent downtime. |
+{: class="hxTable"}
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+## Focus on user goals
+
+Create content that anticipates user needs and explains how a Rackspace feature or option helps the user accomplish a goal. Don't explain the technical details of what a Rackspace service is doing.
+
+For example:
+
+{% figure [caption:"Focus on user goals"] [class:"image bg-light border"] %}
+| Use | Avoid  |
+| --- | --- |
+| Link your existing ObjectRocket account to your Rackspace Cloud account and get easy navigation between control panels. | When you link your existing ObjectRocket account to your Rackspace Cloud account the system provides multiple control panels for a single user ID. |
+{: class="hxTable"}
+{% endfigure %}
+
+Aligning in-product text with user goals requires that you understand the subtleties of your audience and how a feature supports what the user is trying to do.
```

Comment:
this sentence is floating weirdly. is it a part of the section above? is it it's own section? I'm confused...